Tests/BusinessLogic.Tests/UnitTests/LogicTests/PlayedGamesTests/PlayedGameRetrieverTests/SearchPlayedGamesTests.cs

```diff
@@ -146,7 +148,7 @@ public void ItSetsAllTheFields()
             };
             autoMocker.Get<IDataContext>().Expect(mock => mock.GetQueryable<PlayedGame>()).Return(playedGames.AsQueryable());
 
-            var results = autoMocker.ClassUnderTest.SearchPlayedGames(new PlayedGameFilter()).First();
+            var results = autoMocker.ClassUnderTest.SearchPlayedGames(new PlayedGameFilter(), null).First();
```

Comment:
If we aren't testing anything related to ApplicationUser and it isn't required, then I see no reason to pass anything other than null.